server/controller/src/main/java/ai/starwhale/mlops/api/DatasetApi.java

```diff
@@ -144,6 +146,23 @@ ResponseEntity<ResponseMessage<DatasetInfoVo>> getDatasetInfo(
             @RequestParam(value = "versionUrl", required = false)
             String versionUrl);
 
+    @PostMapping(value = "/project/{projectUrl}/dataset/{datasetUrl}/version/{versionUrl}/nextRange")
+    @PreAuthorize("hasAnyRole('OWNER', 'MAINTAINER', 'GUEST')")
+    ResponseEntity<ResponseMessage<DataRangeDesc>> getNextDataRange(
```

Comment:
`getNextDataRange` is technical term, while `consume` is a business term.

Reply:
done